css/interface.css

```diff
@@ -1001,7 +1001,8 @@ header .btn,
   margin-top: 10px;
 }
 
-.image-type-select .selected-folder {
+.image-type-select .selected-folder,
+ .alert-message {
```

Comment:
there's an empty space at the beginning of this line

Reply:
Fixed.